Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,362 @@
+however follow an RRA.
+The RRA focuses on capturing as much information as possible about the risk posture of a service in a short amount of time, typically 30 minutes.
+'''The RRA process analyze and assess services'''. Analyzing processes, or individual pieces of data is not recommended through the RRA process.
+|}
+
+== Preamble ==
+
+Data is the most important item in risk management. Software, websites, infrastructure, networks and people handle, process, exchange and store data.
+
+The RRA focuses efforts on creating a summary of the risks associated with our data. In particular it aims to be:
+
+* '''Quick!''' About 30 minutes to 60 minutes maximum.
+* '''Very high-level'''. Details are for complete threat models.
+* '''Concise, readable'''. Short table with clear risk levels.
+* '''Easy to update'''. Can be run during the architecture phase, and re-run if the project is re-architected or new high level design decisions are made.
+* '''Informative''' It collects risk impact, an approximate impact likelihood and data dictionary.
+
+This helps to make the following type of risk-based decisions:
+
+* Is the security provided by a given platform appropriate to host a specific type of data?
+* How much should we care about maintenance, etc?
+* Does this service warrant a full security threat-model, peneration test, privacy review?
+* How long should we spend on these tasks?
+* Is there anything obvious I should really look at fixing right now?
+
+{| class="wikitable"
+|-
+! <span style="color:gray;">Sample use case</span>
+|-
+|
+Firefox accounts store user data:
+
+* What happens if that data is exposed to the world? What happens if the systems go down? What happens if that data is disclosed? What if the data is modified?
+* Are we at financial, operational risk? Is our reputation also at risk?
+* What kind of data does Firefox accounts exactly handle? How sensitive is the primary kind of data used by Firefox accounts?
+
+The RRA table facilitate the answer to these questions.
+|}
+
+== When to run RRAs? ==
+
+RRAs are designed to be updated and re-runnable as needed. That being said, it is recommended to run the first RRA
+during the design or architecture phase of new services, when an initial data flow diagram for the service has been created.
+
+== What to run through RRAs? ==
+
+RRAs can only be used for services. Any project belongs to or provides a service. Any component, piece of code or data
+belongs to a service. Find which service it belongs to and RRA that service, not the specific piece of code or data.
+
+Large services should be split into multiple smaller services or sub-services that handle a specific type of data and
+expose a limited set of features. This choice has to be made by the security engineer running the RRA. If the
+sub-services belong to different teams, it is a strong indicator that multiple RRAs should be ran.
```

Comment:
I'm not sure on the team distinction; I think at the end of the day the impact remains the same, it might help to include something describing how RRA outcomes for linked services are combined into a realized perspective of risk for a service if it's been split up.